src/Cli/dotnet/Commands/Pack/PackCommandParser.cs

```diff
@@ -108,6 +108,10 @@ private static Command ConstructCommand()
         command.Options.Add(ConfigurationOption);
         command.Options.Add(CommonOptions.DisableBuildServersOption);
         command.Options.Add(TargetOption);
+        command.Options.Add(CommonOptions.GetPropertyOption);
```

Comment:
Does this cause these options to appear in `dotnet pack -h` output?

If so, I don't think this improves customer experience. I feel like it makes the output noisy, for commands that are not pack related.  I expect `-getProperty`, `-getItem`, and so on are listed in `dotnet msbuild` help.